### DISCUSSION TITLE
[Comments] Ability to see recent comments (for ALL documents)

### DISCUSSION THREAD

**Discussion by Shaddix** [outline/outline#5870] [Ideas]
We use comments to leave suggestions to a document author on changing a certain piece of text.
Having quite a lot of documents, it'd be convenient for document authors to see all recent comments (for ALL documents on a single page), so he/she could know what change requests users have left.

Current workaround: use Notifications on new comments - this works, but web UI would be much better.
Another workaround is to go through all pages that the Author has created and check for new comments (which is really inconvenient)

Other good option would be to have an API to retrieve all comments (currently I get the `"message": "documentId: Required"` when requesting `/api/comments.list` without `documentId`.

**Comment by tommoor** [comment]
Totally, we need more views like this. Lets start with the API endpoint – I'll create an issue for that, it's an easy task for anyone.

**Reply by dkrhodes** [reply]
yes. Maybe a watch API is better? So that we can notify people in external system like telegram when comment at someone.

**Reply by tommoor** [reply]
This already exists, you can use webhooks for it 😉

**Comment by tmeuze** [comment]
I love this idea. Maybe a section in _Home_ alongside _Recently viewed_, etc.
